content/06.methods.md

```diff
@@ -452,54 +460,54 @@ Next, we used multivariate cox (proportional hazards) regression analysis [@url:
-|Merged summary files                    |this project                        |https://cavatica.sbgenomics.com/u/cavatica/openpbta                    |
-|Merged summary files and downstream analyses                   |this project                        |https://github.com/AlexsLemonade/OpenPBTA-analysis/                    |
-|Processed data                    |this project                        |https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta                   |
-|Experimental models: Cell lines|||
-|CBTN pediatric brain tumor-derived cell lines                    |[@doi:10.1093/neuonc/noz192]                        |See **Table S1** for identifiers                   |
-|Software and algorithms|||
-|Data processing and analysis software                    |Multiple                        |See **Table S5** for identifiers                    |
-|OpenPBTA workflows repository                   |this project                        |[@doi:10.5281/zenodo.6561352]                    |
-|OpenPBTA analysis repository                    |this project                        |                    |
-|OpenPBTA manuscript repository                    |this project                        |                    |
-|                    |                        |                    |
-|Other|||
-|TCGA WXS dataset                    |National Institutes of Health The Cancer Genome Atlas (TCGA)
-                        |dbGAP phs000178.v11.p8                    |
-|Cancer hotspots                    |MSKCC                        |https://www.cancerhotspots.org/#/download (v2)                    |
-|Reference genomes                    |Broad                        |https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/                    |
-|Reference genome hg38, patch release 12                  |UCSC                        |http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/                    |
-|Human Cytoband file                    |UCSC                        |http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz                     |
-|CDS from GENCODE v27 annotation                |GENCODE                        |https://www.gencodegenes.org/human/release_27.html                     |
-|PFAM domains and locations                    |UCSC                        |http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz; https://pfam.xfam.org/family/PF07714                      |
-|BSgenome.Hsapiens.UCSC.hg38 annotations                   |Bioconductor                        |https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html                     |
-|                    |                        |                     |
+| REAGENT or RESOURCE                           | SOURCE                                                       | IDENTIFIER                                                                                                    |   |
```

Comment:
Something is still funky with table rendering. I think it's too much in the header, because I only see three columns in the table contents. I'm making a suggestion here that I think is right, but I'm not sure!! 

three columns only:

```suggestion
| REAGENT or RESOURCE      | SOURCE   | IDENTIFIER     |
```

Reply:
Are you saying you only see 3 or you should see 3? I did delete the last column since it was all blank and there _should be 3_, I think. Let me check the latest rendering.